web_src/css/markup/content.css

```diff
@@ -128,7 +128,7 @@
 .markup ul,
 .markup ol,
 .markup dl,
-.markup table,
+.markup table:not(.file-preview),
```

Comment:
Instead of all these `:not`, can we instead just use positive matching, e.g. `.markup table.file-preview` to override the conflicting rules?